### ISSUE TITLE
[Bug-App]: "/" in metric name no longer groups into sections

### ISSUE THREAD

**Issue by TheoBoyer** [wandb/wandb#8593]
### Describe the bug

Hi,

When logging metrics in the format section_name/metric_name, instead of grouping the charts under the respective section_name, the charts are appearing under the generic "Charts" section with the metric names displayed as section_name/metric_name. This behavior seems to have changed recently, as I noticed a few people reporting the same problem in #1834 in the last 24 hours.
![Image](https://github.com/user-attachments/assets/4c1a6a2b-644b-496f-951a-a342f97fd251)
It seems to have been broken recently.
Thank you for your time

**Comment by mimbres** [comment]
This is definitely a bug. Suddenly, starting today, the prefix is being ignored, and all graphs are being grouped in the Chart section!

**Comment by mimbres** [comment]
I will!

**Comment by mimbres** [comment]
At first, there was no issue while testing `wandb.log()`. However, when I use it with `lightning` in my code, problems arise. 
```
wandb=0.8.13
pytorch_lightning=2.4.0
torch=2.4.1
```

```python3
import torch
from torch.utils.data import DataLoader, Dataset
from pytorch_lightning import LightningModule, Trainer
from pytorch_lightning.loggers import WandbLogger

class DummyDataset(Dataset):

    def __len__(self):
        return 1

    def __getitem__(self, idx):
        return torch.tensor([0.0]), torch.tensor([0.0])


class BoringModel(LightningModule):

    def __init__(self):
        super().__init__()
        self.layer = torch.nn.Linear(1, 1)
        self.num = 0.

    def forward(self, x):
        return self.layer(x)

    def training_step(self, batch, batch_idx):
        self.log('train_acc', 123)
        self.log('val_acc', 123)
        self.log_dict({"train/loss": self.num, "train/acc": self.num})
        self.log_dict({"validation/loss": self.num, "validation/acc": self.num})
        self.num = self.num + 1
        return None

    def configure_optimizers(self):
        return None

    def train_dataloader(self):
        return DataLoader(DummyDataset(), batch_size=1)

model = BoringModel()
wandb_logger = WandbLogger(log_model="all", project="dummy_project_3", id="exp3", allow_val_change=True)
trainer = Trainer(max_epochs=10, logger=wandb_logger, log_every_n_steps=1)
trainer.fit(model)
```
`validation` and `train` prefixes for section grouping are not working.  

![Image](https://github.com/user-attachments/assets/ac8cdb5c-9842-4cb8-bda4-0016435146c3)

**Comment by fuyutarow** [comment]
I and my colleagues are encountering the same issue. I hope this bug is resolved soon.
```
wandb==0.18.3
lightning==2.4.0
torch==2.4.1
```

**Comment by omerferhatt** [comment]
Any updates? This bug has completely disrupted the UI, making it difficult to read graphs properly.

**Comment by ArtsiomWB** [comment]
Hi everyone,

A quick follow-up from our side. Our engineering team has merged a fix for this on wandb.ai, so you should no longer experience this issue. For W&B server deployments, the fix will be available in the next server version.

Thank you very much for reporting this concern to us and apologies for the inconvenience this has caused.
